TAO/TAO_IDL/ast/ast_expression.cpp

```diff
@@ -2697,9 +2697,11 @@ dump_expr_val (ACE_OSTREAM_TYPE &o, AST_Expression::AST_ExprValue *ev)
     case AST_Expression::EV_char:
       o << ev->u.cval;
       break;
+#ifndef ACE_HAS_CPP20
```

Comment:
Instead of not dumping any value, maybe print something to indicate that we have a non-printable character?

Reply:
Yes, it will do that using the `default:` case.